aggsender/flows/flow_aggchain_prover.go

```diff
@@ -87,7 +90,15 @@ func (a *AggchainProverFlow) CheckInitialStatus(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("aggchainProverFlow - error getting last sent certificate: %w", err)
 	}
-	return a.sanityCheckNoBlockGaps(lastSentCertificate)
+	err = a.sanityCheckNoBlockGaps(lastSentCertificate)
```

Comment:
Do we even care performing a sanity check if it the `requireNoFEPBlockGap` config is set to `false`?
It seems to me that the answer is not straight-forward.